site/src/examples/omezarr/omezarr-viewer.tsx

```diff
@@ -56,6 +57,7 @@ export function OmezarrViewer({
                 numChannels,
                 queueOptions: { maximumInflightAsyncTasks: 2 },
             });
+            // @ts-expect-error we'll deal with this later
```

Comment:
There's a handful of these, I didn't care to diagnose the TS error since it's in Examples and works well

Reply:
hmmm - perhaps my vscode setup is broken? for me, vscode says none of the ts-expects are needed, however the tradeoff is that none of my imports are resolved :(

Reply:
Make sure to do a full `pnpm install` and `pnpm build` to get everything fully up to date if you haven't yet. If that's not working, let's take a closer look together.